src/NUnitFramework/nunitlite.runner/AutoRun.cs

```diff
@@ -86,6 +86,10 @@ public int Execute(string[] args)
                 return TextRunner.OK;
             }
 
+            // We already showed version as a part of the header
+            if (options.ShowVersion)
+                return TextRunner.OK;
+
```

Comment:
You missed the AutoRun in portable. After 3.0 is released, we should see if we can re-merge these files, probably with your branch to wrap the console?